tests/sentry/api/endpoints/test_organization_projects_experiment.py

```diff
@@ -111,6 +112,54 @@ def test_valid_params(self):
         assert project.name == project.slug == self.p1
         assert project.teams.first() == team
 
+    @with_feature(["organizations:team-roles", "organizations:team-project-creation-all"])
+    def test_project_slug_is_slugified(self):
+        unslugified_name = "not_slugged_$!@#$"
+        response = self.get_success_response(
+            self.organization.slug, name=unslugified_name, status_code=201
+        )
+
+        team = Team.objects.get(slug=self.t1, name=self.t1)
+        assert not team.idp_provisioned
+        assert team.organization == self.organization
+        assert team.name == team.slug == self.t1
+
+        member = OrganizationMember.objects.get(user=self.user, organization=self.organization)
+        assert OrganizationMemberTeam.objects.filter(
+            organizationmember=member, team=team, is_active=True, role="admin"
+        ).exists()
+
+        project = Project.objects.get(id=response.data["id"])
+        assert project.name == unslugified_name
+        assert project.slug == slugify(unslugified_name)
+        assert project.teams.first() == team
+
+    @with_feature(["organizations:team-roles", "organizations:team-project-creation-all"])
+    def test_team_slug_is_slugified(self):
+        special_email = fetch_slugifed_email_username("++!#$%&'*+-/=.me@test.com")
+        self.t1 = f"team-{special_email}"
```

Comment:
nit: you don't _have_ to set `self.t1` -- you could just have a local `t1` variable that you check here